dbt_sugar/core/task/base.py

```diff
@@ -230,11 +234,11 @@ def update_column_description_from_schema(
             for column in model.get("columns", []):
                 column_name = column["name"]
                 if column_name in dict_column_description_to_update:
-                    new_desctiption = dict_column_description_to_update[column_name].get(
+                    new_description = dict_column_description_to_update[column_name].get(
```

Comment:
good catch. I tend to be the master of consistent typos since I use autocomplete so much 😂